README.md

```diff
@@ -1,5 +1,57 @@
 # The Interop Project
 
-Welcome to the Interop project, an ongoing effort to make the web more interoperable in key areas, prioritized by web developer and user needs. This is part of [the web-platform-tests project](https://github.com/web-platform-tests/wpt), the main test suite for the web platform.
+Welcome to the Interop Project, an ongoing effort to make the web more interoperable in key areas, prioritized by web developer and user needs. This is part of [the web-platform-tests project](https://github.com/web-platform-tests/wpt), the main test suite for the web platform, and worked on by the [Interop Team](./supporters.md).
 
 Please see the [Interop 2023](./2023/README.md) for the current iteration of the project!
+
+
+## The Purpose of the Interop Project
+
+*Improve interoperability significantly for the benefit of web developers and users.*
+
+The goal of the Interop Project is to improve the web by making it easier to make websites and web apps that work in every browser or browser engine at the same time.
+
+This is done by increasing the amount of “interoperability” between browsers — when each browser engine has implemented the same technology the exact same way, as bug-free as possible.
+
+Today’s browsers have made a commitment to implement web technology according to a shared web standard, created in organizations such as the W3C or WHATWG, where technologies such as CSS and HTML are officially defined.
+
+There is a seemingly infinite amount of work that browser engineering teams could be focused on. The Interop Project provides incentives to focus on the specific and practical work that will help web developers most in the coming year.
```

Comment:
I don't think we should frame this as "help web developers most", but maybe something more generic like "have the most positive impact on the web platform"

Reply:
```suggestion
There is a seemingly infinite amount of work that browser engineering teams could be focused on. The Interop Project provides incentives to focus on the specific and practical work that will have the most positive impact on the web platform in the coming year.
```